packages/dx-react-grid-bootstrap3/src/plugins/table-selection.jsx

```diff
@@ -23,3 +23,5 @@ export class TableSelection extends React.PureComponent {
   }
 }
 
+TableSelection.SelectCell = TableSelectCell;
```

Comment:
Should be called 'Cell' according to https://github.com/DevExpress/devextreme-reactive/pull/502

Reply:
Yes, I saw it. That's why `[wip]` ))